ios/Sources/Model/Blog.swift

```diff
@@ -21,48 +22,40 @@ public struct Blog: Equatable, Identifiable {
         author: Author,
         language: String
     ) {
-        self.feedItem = .init(
-            id: id,
-            image: image,
-            link: link,
-            media: media,
-            publishedAt: publishedAt,
-            summary: summary,
-            title: title
-        )
+        self.id = id
+        self.image = image
+        self.link = link
+        self.media = media
+        self.publishedAt = publishedAt
+        self.summary = summary
+        self.title = title
         self.author = author
         self.language = language
     }
 
     public init(from model: DroidKaigiMPP.FeedItem.Blog) {
-        self.feedItem = .init(
-            id: model.id,
-            image: Image(from: model.image),
-            link: model.link,
-            media: Media.from(model.media),
-            publishedAt: model.publishedAt.toNSDate(),
-            summary: MultiLangText(from: model.summary),
-            title: MultiLangText(from: model.title)
-        )
+        self.id = model.id
+        self.image = Image(from: model.image)
+        self.link = model.link
+        self.media = Media.from(model.media)
+        self.publishedAt = model.publishedAt.toNSDate()
+        self.summary = MultiLangText(from: model.summary)
+        self.title = MultiLangText(from: model.title)
         self.author = Author(from: model.author)
         self.language = model.language
     }
-
-    public subscript<T>(dynamicMember keyPath: KeyPath<FeedItem, T>) -> T {
-        self.feedItem[keyPath: keyPath]
-    }
 }
 
 public extension Blog {
-    var kmmModel: DroidKaigiMPP.FeedItem.Blog {
-        .init(
+    var kmmModel: DroidKaigiMPP.FeedItem {
+        DroidKaigiMPP.FeedItem.Blog(
             id: id,
-            publishedAt: ConvertersKt.toKotlinInstant(feedItem.publishedAt),
-            image: feedItem.image.kmmModel,
-            media: feedItem.media.kmmModel,
-            title: feedItem.title.kmmModel,
-            summary: feedItem.summary.kmmModel,
-            link: feedItem.link,
+            publishedAt: ConvertersKt.toKotlinInstant(publishedAt),
+            image: image.kmmModel,
+            media: media.kmmModel,
+            title: title.kmmModel,
+            summary: summary.kmmModel,
+            link: link,
             language: language,
             author: author.kmmModel
```

Comment:
Why this type changed to `DroidKaigiMPP.FeedItem` ?
I think this `kmmModel` should convert Swift model to appropriate KMM model.
So I think this type should be `DroidKaigiMPP.FeedItem.Blog`

Reply:
I adjusted the interface to achieve your request. ab019c8

And I would also like to ask if we actually need `kmmModel`.
It's required by `FeedRepository`, but the only information it needs is `id` according to the implementation.
https://github.com/DroidKaigi/conference-app-2021/blob/be8c9452d13ef5b6e9121553f2c3027ce53f647a/data/repository/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/FeedRepositoryImpl.kt#L27-L33
So we may adjust the API of `FeedRepository` only to require an `id`, instead of introducing `kmmModel`.
Or if there any other reason that requires `kmmModel`?

Reply:
I see. I don't know about KMM module implementation (It required `id` only ).
I think we should adjust KMM module API interface too.

So let's remove `kmmModel` and use `id` only implementation 🙏🏼

Reply:
I would like to do it in another PR tomorrow since the change is over this PR's scope.

If there is a need to discuss with Android developers or to share that I will do it, please submit an issue and assign it to me. 🙏

Reply:
> If there is a need to discuss with Android developers or to share that I will do it, please submit an issue and assign it to me. 🙏

It's okey!
I think we don't have to submit new issue about that.
So please make Pull Request about that 🙏🏼